source/routing/raptor_solution_reader.cpp

```diff
@@ -74,7 +74,13 @@ struct Journey {
                 sections.emplace_back(elt->end_st, elt->end_dt, elt->begin_st, elt->begin_dt);
             }
         }
-        if (reader.v.clockwise()) { boost::reverse(sections); }
+        if (reader.v.clockwise()) {
+            boost::reverse(sections);
+        } else {
+            // the request is clockwise, thus we need to left align
```

Comment:
I don't think the comment 'the request is clockwise' is very clear since it is in the 'else' of a clockwise test :smile: 

It might be better to remind here that the reader's visitor clockwise is the opposite of the request's clockwise